### DISCUSSION TITLE
A single and a multiple file relation in collection?

### DISCUSSION THREAD

**Discussion by Dumpstar56** [directus/directus#13118] [Archive]
I wanted to test a new feature before dismantling the old one and added a multiple file relation to a collection where there is also a single file relation. But with both relations in the collection, when I click on an item in the table of contents, the message displays that there is nothing there.
Should I be able to have both a single and a multiple file relation in the same collection?

**Comment by rijkvanzanten** [comment]
> Should I be able to have both a single and a multiple file relation in the same collection?

Yup! That shouldn't be an issue. Could be that there's something misconfigured though, but that's hard to tell from the given information 🙂
